internal/utils/main.go

```diff
@@ -240,13 +240,15 @@ func AddCommonFlags(flags *pflag.FlagSet) {
 	flags.Uint32("num-workers", 10, "Number of workers to spawn that read txmeta files from the datastore.")
 	flags.Uint32("retry-limit", 3, "Datastore GetLedger retry limit.")
 	flags.Uint32("retry-wait", 5, "Time in seconds to wait for GetLedger retry.")
+	flags.Bool("write-parquet", true, "If set, write output as parquet files.")
 }
 
 // AddArchiveFlags adds the history archive specific flags: output, and limit
 // TODO: https://stellarorg.atlassian.net/browse/HUBBLE-386 Rename AddArchiveFlags to something more relevant
 func AddArchiveFlags(objectName string, flags *pflag.FlagSet) {
 	flags.Uint32P("start-ledger", "s", 2, "The ledger sequence number for the beginning of the export period. Defaults to genesis ledger")
 	flags.StringP("output", "o", "exported_"+objectName+".txt", "Filename of the output file")
+	flags.String("parquet-output", "exported_"+objectName+".parquet", "Filename of the parquet output file")
```

Comment:
Note parquet-output needs to be defined twice because the refactor work from https://stellarorg.atlassian.net/browse/HUBBLE-386 has not been completed